addons/website_forum/tests/test_controller.py

```diff
@@ -47,6 +47,6 @@ def test_02_admin_attachment(self):
                                                         'mimetype': 'text/plain', 'res_model': 'forum.post',
                                                         'raw': self.pixel, 'website_id': 1}])
         domain = [('name', '=', 'test_pixel')]
-        result = attachment.search_read(domain)
+        result = attachment.search(domain)
```

Comment:
We only use the first record anyway
```suggestion
        result = attachment.search(domain, limit=1)
```

Reply:
FYI, sometime, adding a `limit 1` can change drastically the SQL plan.
But we are in a test case, then we don't care about this. (I made the change :+1: )